modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/CoreStatistics_conf.pm

```diff
@@ -130,38 +130,65 @@ sub pipeline_analyses {
                          },
       -max_retry_count => 1,
       -flow_into       => {
-                            '2' => ['GeneRelatedTasks'],
+                            '2' => ['CheckAssemblyGenesetGeneTasks'],
                           },
       -rc_name         => 'normal',
     },
-
+    { -logic_name  => 'CheckAssemblyGenesetGeneTasks',
+      -module      => 'Bio::EnsEMBL::Production::Pipeline::Common::CheckAssemblyGeneset',
+      -parameters  => {
+          run_all => $self->o('run_all'),
+          release => $self->o('release')
+       },
+      -can_be_empty    => 1,
```

Comment:
A minor point, but I don't think this line is required.

Reply:
I've removed this.